lib/studies/reply/app.dart

```diff
@@ -17,6 +18,38 @@ class ReplyApp extends StatelessWidget {
 
   static const String homeRoute = '/reply';
   static const String searchRoute = '/reply/search';
+  static const String composeRoute = '/reply/compose';
+
+  static Route createSearchRoute() {
+    return PageRouteBuilder<void>(
+      pageBuilder: (context, animation, secondaryAnimation) =>
+          const SearchPage(),
+      transitionsBuilder: (context, animation, secondaryAnimation, child) {
+        return SharedAxisTransition(
+          fillColor: Theme.of(context).cardColor,
+          animation: animation,
+          secondaryAnimation: secondaryAnimation,
+          transitionType: SharedAxisTransitionType.scaled,
+          child: child,
+        );
+      },
+    );
+  }
+
+  static Route createComposeRoute() {
+    return PageRouteBuilder<void>(
+      pageBuilder: (context, animation, secondaryAnimation) =>
+          const ComposePage(),
+      transitionsBuilder: (context, animation, secondaryAnimation, child) {
+        return FadeThroughTransition(
```

Comment:
Why is the compose route a FadeThrough but the search route is a SharedAxis transition? Is that the expected behavior?

Reply:
This is expected behavior, the search route always used SharedAxis transition. Only the compose route is changing.